testnets/archwaytestnet/assetlist.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "constantine-1",
```

Comment:
please change this chain_name to `archwaytestnet` the chain_id will still be retained in the chain.json file.

Reply:
Fixed